code/src/WijmoProvider/Features/ColumnSort.ts

```diff
@@ -214,34 +231,58 @@ namespace WijmoProvider.Feature {
             });
         }
 
+        /**
+         * Function that sorts a Grid column based in its ID and on a sorting
+         * @param  {string} columnID
+         * @param {OSFramework.OSStructure.Sorting} sorting
+         * @returns {*}  {OSFramework.OSStructure.ReturnMessage} Return Message containing the resulting code from sorting columns and the error message in case of failure
+         */
         public sortColumn(
             columnID: string,
             sorting: OSFramework.OSStructure.Sorting
-        ): void {
-            const column = this._grid.getColumn(columnID);
-            const ascending =
-                OSFramework.OSStructure.Sorting[sorting] ===
-                OSFramework.OSStructure.Sorting.Ascending;
+        ): OSFramework.OSStructure.ReturnMessage {
+            try {
+                const column = this._grid.getColumn(columnID);
+                const ascending =
+                    OSFramework.OSStructure.Sorting[sorting] ===
+                    OSFramework.OSStructure.Sorting.Ascending;
```

Comment:
In what situations does an exception occur?

Reply:
When we pass for instance an invalid column ID.

Reply:
If the column doesn't exist, it will not reach here, right @glundgren93 ?

Reply:
Just chatted with Gabriel. Please proceed.